css/index.css

```
@@ -112,6 +148,107 @@ p {
     justify-content: center;
 }
 
+.box1{
```

```suggestion
.box{
	width: 12.5%;
    height: 100px;
    margin: 20px 2.5%;
    color: white;
    display: flex;
    align-items: center;
    justify-content: center;
}

.box1 {
	background:teal;
}

.box2 {
	background:gold;
}
```
Continue the pattern